README.rst

```diff
@@ -61,6 +61,165 @@ Application requirements:
+
+    wget http://installer.host/setup/ | sudo bash
+
+2.- Install monitor:
+
+request::
+
+    curl -d '{"hosts": ["mon1.host"], "redhat_storage": true}' -X POST http://installer.hosts/api/mon/install/
+
+response::
+
+    {
+        "endpoint": "/api/mon/install/",
+        "succeeded": false,
+        "stdout": null,
+        "started": null,
+        "exit_code": null,
+        "ended": null,
+        "command": null,
+        "stderr": null,
+        "identifier": "47f60562-a96b-4ac6-be07-71726b595793"
+    }
+
+Note: the identifier can be immediately used get task metadata information::
+
+    curl http:://installer.host/api/tasks/47f60562-a96b-4ac6-be07-71726b595793/
+    {
+        "endpoint": "/api/mon/install/",
+        "succeeded": false,
+        "stdout": null,
+        "started": "2016-02-15 14:24:06.414728",
+        "exit_code": null,
+        "ended": null,
+        "command": "/usr/local/bin/ansible-playbook /tmp/ceph-ansible/site.yml -i /var/folders/t8/smzdykh12h39f8r0vwv5vzf00000gn/T/47f60562-a96b-4ac6-be07-71726b595793__ilpiv --extra-vars {\"ceph_stable\": true} --tags package-install",
+        "stderr": null,
+        "identifier": "47f60562-a96b-4ac6-be07-71726b595793"
+    }
+
+3.- Install OSDs:
+
+request::
+    curl -d '{"hosts": ["osd1.host", "osd2.host"], "redhat_storage": true}' -X POST http://installer.hosts/api/osd/install/
+
+response::
+
+    {
+        "endpoint": "/api/osd/install/",
+        "succeeded": false,
+        "stdout": null,
+        "started": null,
+        "exit_code": null,
+        "ended": null,
+        "command": null,
+        "stderr": null,
+        "identifier": "47f60562-a96b-4ac6-be07-71726b595793"
+    }
+
+
+Task metadata for the previous request is then available at::
+
+    curl http:://installer.host/api/tasks/03965afd-6ae3-40e5-9530-3ac677a43226/
+
+
+4.- Configure monitor:
+
+request::
+    curl -d '{"host": "mon1.host", "monitor_interface": "eth0", "fsid": "deedcb4c-a67a-4997-93a6-92149ad2622a"}' -X POST http://installer.hosts/api/osd/install/
+
+response::
+
+    {
+        "endpoint": "/api/mon/configure/",
+        "succeeded": false,
+        "stdout": null,
+        "started": null,
+        "exit_code": null,
+        "ended": null,
+        "command": null,
+        "stderr": null,
+        "identifier": "4fe75438-1c76-40f9-b39c-9dbe78af28ed"
+    }
+
+Task metadata for the previous request is then available at::
+
+    curl http:://installer.host/api/tasks/4fe75438-1c76-40f9-b39c-9dbe78af28ed/
+
+
+4.- Configure OSDs:
+Note that we are using ``journal_collocate`` flag to indicate we are going to
+collocate the journal in the same device as the OSD. This is *not ideal* and
+*not recommended for production use*, but it makes example setups easier to
+describe.
+
+request::
+    curl -d '{"host": "osd1.host", "devices": ["/dev/sdb/"], "journal_collocate": true, "fsid": "deedcb4c-a67a-4997-93a6-92149ad2622a"}' -X POST http://installer.hosts/api/osd/configure/
```

Comment:
I'm thinking we might need some more POST data here to actually configure an OSD, but seeing as that we haven't completed that endpoint yet I think it's safe to leave this as-is and update later.